src/app/theme.ts

```diff
@@ -0,0 +1,38 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+
+// Theme documentation: https://material-ui.com/customization/palette/
+// Theme demo: https://material.io/resources/color/#!/?view.left=1&view.right=1&primary.color=003365&secondary.color=FBC02D
+// Do not export this directly! theme should be accessed within makeStyles & withStyles. Use ErrorMessage.tsx as a reference
+export const theme = createMuiTheme({
```

Comment:
The comment says not to export?

Reply:
Yeah, I had to make it exportable so that I can use the same theme in Storybook `.storybook/decorators/ThemeDecorator.tsx`. I will change the wording to `Do not export this directly for styling!`